src/GlobalStyles.js

```diff
@@ -46,6 +47,6 @@ textarea {
   letter-spacing: 1px;
   font-size: 18px;
   line-height: 1.5;
-  color: var(--lightgray);
+  color: var(--text-main);
 }
```

Comment:
Nice Celine ;) You could save some code for the same things...if you write for e.g. `input, textarea {background-color: transparent; color: var(--text-main); font-family: 'Special Elite';    letter-spacing: 1px; font-size: 18px;` and then for input and for textarea just the things which are different